ingestion/README.md

```diff
@@ -0,0 +1,52 @@
+# Ingestion
+
+The `ingestall.py` script is designed to parse and process datasets and load data into a SQLite database. The script offers several command-line flags for customization:
+
+### Command-line Flags
+
+- **`--root-dir`**
+   - **Description**: Specifies the path to the directory containing the datasets to be ingested.
+   - **Type**: String
+   - **Required**: Yes
+
+- **`--dry-run`**
+   - **Description**: If set, the script will undergo all processing steps except for the data ingestion into the SQLite database. Useful for validating and testing the dataset without making actual changes to the database.
+   - **Type**: Flag
+   - **Required**: No
+   - **Default**: False
+
+- **`--transform`**
+   - **Description**: If set, the script will perform additional transformation tasks, specifically zipping the cphates datasets and converting AVI files to MP4 format.
+   - **Type**: Flag
+   - **Required**: No
+   - **Default**: False
+
+- **`--output-dir`**
+   - **Description**: Specifies the directory where output files, such as logs and exported data, should be saved. If the directory doesn't exist, the script will create it.
+   - **Type**: String
+   - **Required**: No
+   - **Default**: 'output'
+
+- **`--db-path`**
+   - **Description**: Specifies the path to the SQLite database file where data will be ingested. If the script is run with the `--dry-run` flag, this database won't be modified.
+   - **Type**: String
+   - **Required**: No
+   - **Default**: '../../applications/neuroscan/backend/.tmp/data.db'
+
+## Dataset Validation
+
+1. Install the necessary packages:
+
+ ```
+pip install -r requirements.txt
+```
+
+2. Make sure python knows where to look up for the modules:
+```
+export PYTHONPATH=$PYTHONPATH:/path/to/geppetto-NeuroSCAN
```

Comment:
suggestion:: wouldn't it be nicer to add the geppetto-NeuroSCAN package to the python path using `pip install`? @afonsobspinto

Reply:
:+1: